src/opensles/AudioStreamBuffered.cpp

```diff
@@ -25,23 +25,18 @@ namespace oboe {
  * AudioStream with a FifoBuffer
  */
 AudioStreamBuffered::AudioStreamBuffered(const AudioStreamBuilder &builder)
-        : AudioStream(builder)
-        , mFifoBuffer(nullptr)
```

Comment:
Why not to change the field type to a smart pointer and forget about manual initialization / deletion:

std::unique_ptr<FifoBuffer> mFifoBuffer;

Reply:
Done